delfin/task_manager/manager.py

```diff
@@ -16,18 +16,28 @@
 **periodical task manager**
 
 """
-
+import json
+from datetime import datetime
+from apscheduler.schedulers.blocking import BlockingScheduler
 from oslo_config import cfg
 from oslo_log import log
 from oslo_utils import importutils
 
-from delfin import manager
+from delfin import manager, exception
+from delfin.api.v1.storages import StorageController
 from delfin.drivers import manager as driver_manager
 from delfin.task_manager.tasks import alerts
 
 LOG = log.getLogger(__name__)
 CONF = cfg.CONF
-CONF.import_opt('periodic_interval', 'delfin.service')
+
+MIN_INTERVAL = 5
```

Comment:
Can we move it inside some only constants file?

Reply:
done